app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationScreen.kt

```diff
@@ -153,84 +148,184 @@ fun CompleteRegistrationScreen(
+                textAlign = TextAlign.Center,
             )
             Spacer(modifier = Modifier.height(8.dp))
-            PasswordStrengthIndicator(
-                modifier = Modifier.padding(horizontal = 16.dp),
-                state = state.passwordStrengthState,
-            )
-            Spacer(modifier = Modifier.height(8.dp))
-            BitwardenPasswordField(
-                label = stringResource(id = R.string.retype_master_password),
-                value = state.confirmPasswordInput,
-                showPassword = showPassword,
-                showPasswordChange = { showPassword = it },
-                onValueChange = remember(viewModel) {
-                    { viewModel.trySendAction(ConfirmPasswordInputChange(it)) }
-                },
-                modifier = Modifier
-                    .testTag("ConfirmMasterPasswordEntry")
-                    .fillMaxWidth()
-                    .padding(horizontal = 16.dp),
-                showPasswordTestTag = "ConfirmPasswordVisibilityToggle",
-            )
-            Spacer(modifier = Modifier.height(16.dp))
-            BitwardenTextField(
-                label = stringResource(id = R.string.master_password_hint),
-                value = state.passwordHintInput,
-                onValueChange = remember(viewModel) {
-                    { viewModel.trySendAction(PasswordHintChange(it)) }
-                },
-                hint = stringResource(id = R.string.master_password_hint_description),
-                modifier = Modifier
-                    .testTag("MasterPasswordHintLabel")
-                    .fillMaxWidth()
-                    .padding(horizontal = 16.dp),
-            )
-            Spacer(modifier = Modifier.height(24.dp))
-            BitwardenSwitch(
-                label = stringResource(id = R.string.check_known_data_breaches_for_this_password),
-                isChecked = state.isCheckDataBreachesToggled,
-                onCheckedChange = remember(viewModel) {
-                    { newState ->
-                        viewModel.trySendAction(CheckDataBreachesToggle(newState = newState))
-                    }
-                },
-                modifier = Modifier
-                    .testTag("CheckExposedMasterPasswordToggle")
-                    .padding(horizontal = 16.dp),
+            Text(
+                text = "Choose a unique and strong password to keep your information safe.",
+                style = MaterialTheme.typography.bodyMedium,
+                color = MaterialTheme.colorScheme.onSurface,
+                textAlign = TextAlign.Center,
             )
-            Spacer(modifier = Modifier.navigationBarsPadding())
         }
     }
+
+    if (configuration.orientation == Configuration.ORIENTATION_PORTRAIT) {
+        Column(
+            modifier = modifier,
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            orderedContent()
+        }
+    } else {
+        Row(
```

Comment:
Wanna add a test that uses `@Config(qualifiers = "land")` just to make this gets hit.